packages/parchment/lib/src/codecs/html.dart

```diff
@@ -551,10 +551,17 @@ class _ParchmentHtmlEncoder extends Converter<ParchmentDocument, String> {
           return;
         }
         if (embeddable.type == 'image') {
-          // Force the image to fit within any max. width that might be set. If
-          // no width or max-width is set on an outer block, then this does nothing.
-          buffer.write(
-              '<img src="${embeddable.data['source']}" style="max-width: 100%; object-fit: contain;">');
+          final attributes = StringBuffer();
```

Comment:
This introduces a breaking change. Can we fall back on previous style when no data available (e.g.: `style="max-width: 100%; object-fit: contain;`)?

Reply:
@marcio-ota It doesn't seem you have addresses this comment